centaur/src/main/resources/standardTestCases/retry_with_more_memory/do_not_retry_rc0.wdl

```diff
@@ -0,0 +1,21 @@
+version 1.0
+
+task imitate_oom_error {
+  command {
+    # This task mimics printing an exception that would cause it to retry with increased memory, but
+    # because the return code is 0 (success) the task does not retry.
+    printf "Exception in thread "main" java.lang.OutOfMemoryError: testing\n\tat Test.main(Test.java:1)\n" >&2 && (exit 0)
+  }
+  output {
```

Comment:
I am not including the "foo" output file because that throws an error (since it does not exist). There is a test case related to that-- retry_same_memory_output_failure.